src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  AppBar,
+  Toolbar,
+  Typography,
+  IconButton,
+  Button,
+  Menu,
+  MenuItem,
+} from '@mui/material';
+import NotificationsNoneIcon from '@mui/icons-material/NotificationsNone';
+import SearchIcon from '@mui/icons-material/Search';
+import MenuIcon from '@mui/icons-material/Menu';
+import { InputWithIcon } from '../InputWithIcon/InputWithIcon';
+
+const dummyCategories = ['HandBags', 'Watches', 'Skincare', 'Jewllery', 'Apparels'];
+
+// eslint-disable-next-line react/prop-types, no-unused-vars
+export default function Header({ searchPlaceHolder, categories }) {
```

Comment:
please make sure to be consistent in the code syntax. 
if you want to use arrow functions syntax it should be used all over the project.
I'm noticing that you are using both the traditional function syntax declaration here and the arrow function declaration in the rest of this file.